cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -232,13 +237,19 @@ function enableInstantSearch(config) {
                     showMoreText: algoliaData.strings.moreResults,
                     empty: '',
                     item(hit, { html, components }) {
+                        const displayCalloutMsg = !hit.calloutMsg && 'd-none';
+                        const productId = algoliaData.recordModel === 'master-level' ? (hit.defaultVariantID ? hit.defaultVariantID : hit.objectID) : hit.objectID;
+                        const callOutMsgClassname = `callout-msg-placeholder-${productId}`;
```

Comment:
What is this class used for?

Reply:
I forget to remove it after a refactoring